Makefile

```diff
@@ -0,0 +1,5 @@
+all: data
```

Comment:
I dream of the day of a better makefile when we aren't doing scripting in bash. But hey it works and its standard 😄

Reply:
There are things like [`just`](https://github.com/casey/just) and others but is hard to beat `make` in terms of availability anywhere you want to run it.